edge-tool/test_data/mbed_cloud_dev_credentials.c

```diff
@@ -0,0 +1,224 @@
+/*
```

Comment:
Are we leaking anything with these files?

Reply:
I had hoped comments on line 4-6 would be sufficient cover that.
No, we're not leaking anything.

Reply:
oh yes - didn't really read the file, just raised the question just in case.